requirements.csv

```diff
@@ -17,3 +17,4 @@ srossross_not_in_db,rpmfile
 indygreg_not_in_db,zstandard
 nir0s_not_in_db,distro
 tiran_not_in_db,defusedxml
+python,requests
```

Comment:
It's [good practice](https://unix.stackexchange.com/a/18789) to add a newline at the end of the file.

Reply:
Thanks for the suggestion! I'll make the edit.